src/services/ContentService.js

```diff
@@ -1108,7 +1108,12 @@ define(["structures/ContentCreateStruct", "structures/ContentUpdateStruct", "str
                     callback(error, views);
                     return;
                 }
-
+                if ( viewCreateStruct.getCriteria() && Object.keys(viewCreateStruct.getCriteria()).length !== 0 ) {
+                    console.warn('[DEPRECATED] virtual property Criteria is deprecated');
+                } else {
+                    viewCreateStruct.setCriteria(null);
```

Comment:
it's really weird to have to do this here. I guess it's because you get an error 500 when sending an empty `Criteria` list. I think @bdunogier proposed to filter such case server side. @StephaneDiot @bdunogier could you have a look at that together ?